test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    @Test
+    public void isAnyNull() throws Exception {
+        // no object
+        assertNoNull();
+        
+        // only one object null
+        assertIsNull((Object) null);
+        
+        // only one object not null
+        assertNoNull(1);
+        assertNoNull("abc");
+        assertNoNull("");
+        
+        // many objects with null
+        assertIsNull(null, 1, "abc");
+        assertIsNull("", "ABC", null, 1, null);
+        assertIsNull(null, null, null);
+        
+        // many objects without null
+        assertNoNull(1, "abc");
+        assertNoNull("", "ABC", 1);
+    }
+    
+    private void assertIsNull(Object... objects) {
```

Comment:
Perhaps this method can be renamed as `assertHasNull` since a list of objects may contain other objects that are not null even though there is one that is null.